packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -87,19 +97,28 @@ enum BottomNavigationBarType {
 /// Widget build(BuildContext context) {
 ///   return Scaffold(
 ///     appBar: AppBar(
-///       title: Text('BottomNavigationBar Sample'),
+///       title: const Text('BottomNavigationBar Sample'),
 ///     ),
 ///     body: Center(
 ///       child: _widgetOptions.elementAt(_selectedIndex),
 ///     ),
 ///     bottomNavigationBar: BottomNavigationBar(
-///       items: <BottomNavigationBarItem>[
-///         BottomNavigationBarItem(icon: Icon(Icons.home), title: Text('Home')),
-///         BottomNavigationBarItem(icon: Icon(Icons.business), title: Text('Business')),
-///         BottomNavigationBarItem(icon: Icon(Icons.school), title: Text('School')),
+///       items: const <BottomNavigationBarItem>[
+///         BottomNavigationBarItem(
+///           icon: Icon(Icons.home),
+///           title: Text('Home'),
+///         ),
+///         BottomNavigationBarItem(
+///           icon: Icon(Icons.business),
+///           title: Text('Business'),
+///         ),
+///         BottomNavigationBarItem(
+///           icon: Icon(Icons.school),
+///           title: Text('School'),
+///         ),
 ///       ],
 ///       currentIndex: _selectedIndex,
-///       selectedItemColor: Colors.deepPurple,
+///       selectedItemColor: Colors.amber[800],
 ///       onTap: _onItemTapped,
 ///     ),
 ///   );
```

Comment:
uber-nit, that you may want to ignore since not really related to your PR: Consider fixing the "See also:" section below this. For each bullet it should include a brief explanation why people may want to look there.